### ISSUE TITLE
Crash while logging bigger string than 256

### ISSUE THREAD

**Issue by ribeirodanielf** [leifclaesson/LeifHomieLib#4]
Hi,

While working with a settable property the code crash every time a message is received and the problem is on the DebugPrint definition.
Instead of use sprint, please consider to use **snprint(szTemp, 255**, __VA_ARGS__);

#define csprintf(...) { char szTemp[256]; sprintf(szTemp,__VA_ARGS__ ); HomieLibDebugPrint(szTemp); }

Regards,
Daniel

**Comment by leifclaesson** [comment]
Done! Thanks for noticing.
I've also rewritten the library to use PangolinMQTT (rather than AsyncMqttClient) since last time you saw it. It works a whole lot better. Turns out all the reliability issues were just bugs in AsyncMqttClient. I'll eventually streamline the library further now that the many workarounds I created in order not to overload AsyncMqttClient with a high message rate on startup, are probably no longer needed.
